packages/ckeditor5-ui/src/dropdown/utils.ts

```diff
@@ -129,11 +132,85 @@ export function createDropdown(
 		buttonView.bind( 'isOn' ).to( dropdownView, 'isOpen' );
 	}
 
-	addDefaultBehavior( dropdownView );
+	addDefaultBehaviors( dropdownView );
 
 	return dropdownView;
 }
 
+/**
+ * Adds an instance of {@link module:ui/dropdown/menu/dropdownmenurootlistview~DropdownMenuRootListView} to a dropdown and sets all
+ * common behaviors and interactions between the dropdown and the menu component.
+ *
+ * Use this helper to create dropdown nested menus that can be added to a toolbar.
+ *
+ * ```ts
+ * const definitions = [
+ * 	{
+ * 	    id: 'menu_1',
+ * 	    menu: 'Menu 1',
+ * 	    children: [
+ * 	        {
+ * 	            id: 'menu_1_a',
+ * 	            label: 'Item A'
+ * 	        },
+ * 	        {
+ * 	            id: 'menu_1_b',
+ * 	            label: 'Item B'
+ * 	        }
+ * 	    ]
+ * 	},
+ * 	{
+ * 	    id: 'top_a',
+ * 	    label: 'Top Item A'
+ * 	},
+ * 	{
+ * 	    id: 'top_b',
+ * 	    label: 'Top Item B'
+ * 	}
+ * ];
+ *
+ * const dropdownView = createDropdown( editor.locale );
+ * const dropdownMenuRootListView = new DropdownMenuRootListView( editor.locale, editor.ui.view.body, definitions );
+ *
+ * addMenuToDropdown( dropdownView, dropdownMenuRootListView );
+ * ```
+ *
+ * After using this helper, the `dropdown` will fire {@link module:ui/dropdown/dropdownview~DropdownViewEvent `execute`} event when
+ * a nested menu button is pressed.
+ *
+ * The helper will make sure that the `dropdownMenuRootListView` is lazy loaded, i.e., the menu component structure will be initialized
+ * and rendered only after the `dropdown` is opened for the first time.
+ *
+ * @param dropdownView A dropdown instance to which the menu component will be added.
+ * @param dropdownMenuRootListView A menu component to be added to the dropdown.
+ */
+export function addMenuToDropdown( dropdownView: DropdownView, dropdownMenuRootListView: DropdownMenuRootListView ): void {
```

Comment:
Pass `definitions` and `body` instead of `dropdownMenuRootListView` - let's hide the complexity from the integrator.

Reply:
Let's set `dropdownView.menuView = dropdownMenuRootListview` to be consistent with other helpers.